docs/table/indexing.rst

```diff
@@ -208,6 +208,59 @@ or table is copied::
   ...    print(t2.indices)
   []
 
+Updating row using indices
+==========================
+
+Row updation can be accomplished by assigning the table property: `~astropy.table.Table.loc` a complete row or a list of rows.::
+
+   >>> t = Table([(1, 2, 3, 4), (10, 1, 9, 9)], names=('a', 'b'), dtype=['i8', 'i8'])
```

Comment:
Although there is precedent from the other examples, I think it is much more clear to have the key value column be a string rather than an int.  This makes it more obvious that the indexing is not normal numpy indexing but rather key indexing.  So use `t = Table([('w', 'x', 'y', 'z'), (10, 1, 9, 9)], ...)`.

Use the same strategy for tests.